src/devices/src/legacy/rtc.rs

```diff
@@ -6,7 +6,7 @@ use vm_device::bus::MmioAddress;
 use vm_device::MutDeviceMmio;
 use vm_superio::{rtc_pl031::NoEvents, Rtc};
 
-use utils::debug;
```

Comment:
The debug macro was doing something special, is this one working the same way? Specifically, we don't want to logs to be filled with data coming from the guest when we have release builds because that can be used to generate a DoS when the log file is not properly rate limited.